app/addons/documents/shared-views.js

```diff
@@ -160,12 +160,21 @@ function(app, FauxtonAPI, Components, Documents, Databases) {
     className:  "nav nav-list",
     template: "addons/documents/templates/design_doc_menu",
     events: {
-      "click .js-collapse-toggle": "toggleArrow"
+      "click .js-collapse-toggle": "toggleArrow",
+      'click .js-toggle-submenu': 'toggleSubmenu'
     },
 
     toggleArrow:  function(e){
-      this.$(e.currentTarget).toggleClass("down");
+      this.$(e.currentTarget).toggleClass('down');
+      $('#' + this.model.id.replace(/^_design\//, '').replace(/\./g, '\\.')).collapse('toggle'); 
```

Comment:
we use `this.$`.

the fix does not work for me:

![bildschirmfoto 2015-03-18 um 15 58 21](https://cloud.githubusercontent.com/assets/298166/6711499/b83793aa-cd87-11e4-8dce-a40a40c1767e.png)